src/lib/storage/disk_analyzer.rb

```diff
@@ -71,21 +72,53 @@ class DiskAnalyzer
 
       DEFAULT_DISK_CHECK_LIMIT = 10
 
-      attr_reader :installation_disks, :candidate_disks
-      attr_reader :windows_partitions, :linux_partitions, :efi_partitions
+      # @return [Array<String>] device names of installation media.
+      #       Filled by #analyze.
+      attr_reader :installation_disks
+
+      # @return [Array<String>] device name of disks to install on.
+      #       Filled by #analyze.
+      attr_reader :candidate_disks
+
+      # @return [Hash] Linux partitions found in each candidate disk.
```

Comment:
+1 for improving documentation for those accessors, but
maybe add [Hash<storage::Partition>] (or whatever it is) for clarification?